src/pybind/mgr/diskprediction/agent/metrics/ceph_pool.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+import socket
+
+from . import MetricsAgent
+from ...common.db import DB_API
+from ...models.metrics.dp import Ceph_Pool
+
+
+class CephPool_Agent(MetricsAgent):
+    measurement = 'ceph_pool'
+
+    def _collect_data(self):
+        # process data and save to 'self.data'
+        obj_api = DB_API(self._ceph_context)
+        df_data = obj_api.get('df')
+        cluster_id = obj_api.get_cluster_id()
+        for pool in df_data.get("pools", []):
+            d_pool = Ceph_Pool()
+            p_name = pool.get("name")
+            p_id = pool.get("id")
+            d_pool.tags['cluster_id'] = cluster_id
+            d_pool.tags['pool_id'] = p_id
+            d_pool.fields['agenthost'] = socket.gethostname()
+            d_pool.tags['agenthost_domain_id'] = \
+                "%s_%s" % (cluster_id, d_pool.fields['agenthost'])

```

Comment:
can we remove the agenthost* fields?

Reply:
Agenthost is for management purpose. It is crucial for our cloud system.